org.eclipse.lyo.repackaging.jersey/pom.xml

```diff
@@ -0,0 +1,130 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.eclipse.lyo.oslc4j.core</groupId>
+	<artifactId>repackaging.jersey</artifactId>
+	<version>2.26</version>

```

Comment:
do we want to use Lyo versioning here instead?

Reply:
I think, the Jersey version required here is 2.26. So if we change it here to the Lyo version, make sure to set jersey version correctly.